user_guide_src/source/libraries/caching.rst

```diff
@@ -36,14 +36,14 @@ $handler
 ========
 
 The is the name of the handler that should be used as the primary handler when starting up the engine.
-Available names are: dummy, file, memcached, redis, predis, wincache.
+Available names are: dummy, file, memcached, redis, predis, wincache. This is commonly the **File** handler
+since the file system is always available, but may not fit more complex, multi-server setups.
 
 $backupHandler
 ==============
 
 In the case that the first choice ``$handler`` is not available, this is the next cache handler to load.
-This is commonly the ``File`` handler since the file system is always available, but may not fit
-more complex, multi-server setups.
```

Comment:
Why do you remove this?

Reply:
@kenjis Because in the code backupHandler is 'dummy'.

Reply:
It is just the default value is `dummy`.
Revert this change or update the description.
It explains `file` is often used for backup.